packages/delegate/src/index.ts

```diff
@@ -9,4 +9,4 @@ export * from './resolveExternalValue';
 export * from './subschemaConfig';
 export * from './transforms';
 export * from './types';
-export { Executor, AsyncExecutor, SyncExecutor, Request } from '@graphql-tools/utils';
+export { Executor, AsyncExecutor, SyncExecutor, ExecutionRequest as Request } from '@graphql-tools/utils';
```

Comment:
Why as Request? Let’s go for breaking changes when we can?

Reply:
Oh IDE :) Fixing it now